database/factories/LegacyMatch/GameFactory.php

```diff
@@ -3,29 +3,37 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
 // See the LICENCE file in the repository root for full licence text.
 
+declare(strict_types=1);
+
+namespace Database\Factories\LegacyMatch;
+
+use App\Models\Beatmap;
 use App\Models\LegacyMatch\Game;
 use Carbon\Carbon;
+use Database\Factories\Factory;
+
+class GameFactory extends Factory
+{
+    protected $model = Game::class;
+
+    public function definition(): array
+    {
+        return [
+            'beatmap_id' => Beatmap::factory(),
```

Comment:
Come to think of it, the keys should probably be sorted while we're updating these and move the ones with dependencies to the end (same for the others) 🤔